lms/services/group_info.py

```diff
@@ -43,7 +43,7 @@ def upsert(self, h_group, consumer_key, params):
         :param h_group: the group to upsert
         :type h_group: models.HGroup
 
-        :param consumer_key: the GroupInfo.consumer_key value to set
+        :param application_instance: the ApplicationInstance this group belongs to
 
         :param params: the other GroupInfo columns to set
         :type params: dict
```

Comment:
It would be good to tidy these type declarations into annotations. Also the reference to `models.GroupInfo` is weird, as it's `GroupInfo` locally.